(/custom-flows/email-password-mfa) Update email password mfa custom flow guide (#1015)
clerk/clerk-docs: docs/custom-flows/overview.mdx

### Factor two verification (optional)

> This step only applies to users that have turned on two-factor authentication, also known as [multifactor authentication](/docs/custom-flows/email-password-mfa), for their user.

Reviewer: ```suggestion
This step only applies to users that have enabled [multifactor authentication](/docs/custom-flows/email-password-mfa) for their user.
```
Let's just call it what it is in the UI here

> When one form of verification isn't enough, trust two! Forcing two different verification steps vastly increases the security of your account. The most common setup a user will have to protect their account is using a password as their first kind of verification and a `phone_code` as their second kind of verification.

Reviewer: ```suggestion
Forcing two different verification steps vastly increases the security of your account. The most common setup a user will have to protect their account is using a password as their first kind of verification and a `phone_code` as their second kind of verification.
```
I know you didn't alter this line, but let's just cut the first sentence in this paragraph as it's just fluff.
